cmake/Modules/FindIconv.cmake

```diff
@@ -36,7 +36,7 @@ IF(ICONV_FOUND)
   #include <iconv.h>
   int main(){
     iconv_t conv = 0;
-    const char* in = 0;
+    char* in = 0;
```

Comment:
This looks wrong. The entire point of this code is to check if having that variable as `const` is valid or not.